cli/pkg/tree/demo/cleanup/appmesh_eks_cleanup.go

```
@@ -0,0 +1,35 @@
+package cleanup
+
+import (
+	"fmt"
+
+	"github.com/solo-io/service-mesh-hub/cli/pkg/common/exec"
+)
+
+func AppmeshEksCleanup(runner exec.Runner, region string, meshName string, eksClusterName string) error {
+	return runner.Run("bash", "-x", "-c", fmt.Sprintf(appmeshEksCleanupScript, region, meshName, eksClusterName))
+}
+
+const (
+	appmeshEksCleanupScript = `
+region=%s
+meshName=%s
+eksClusterName=%s
+awsAccountID=$(echo $(aws sts get-caller-identity --query 'Account'))
+
+if [ -z ${awsAccountID+x} ]; 
+then echo "Unable to fetch AWS account ID, check that your AWS credentials are configured properly." && exit 0 ; 
```

should be exiting nonzero here

---

Exiting non-zero causes a verbose `meshctl` error message to be outputted, and I don't expect this cmd to be used in an automated fashion, so I figured it'd be better to hide the verbosity

---

making non-zero and hiding the error message